utils/utils.go

```diff
@@ -208,6 +208,11 @@ func PopulateEndpointNets(wep *api.WorkloadEndpoint, result *current.Result) err
 	return nil
 }
 
+type CNITestArgs struct {
+	cnitypes.CommonArgs
```

Comment:
It probably also don't need the `CommonArgs` bit since we already unmarshal those elsewhere.

Reply:
https://github.com/containernetworking/cni/blob/master/pkg/types/args.go#L54

It seems we have to embed this CommonArgs into the structure.